src/Defer/Defer.php

```diff
@@ -118,7 +118,7 @@ public function defer(Closure $resolver, string $path)
             return $data;
         }
 
-        if ($this->isDeferred($path) || ! $this->acceptFurtherDeferring) {
+        if (! $this->acceptFurtherDeferring || $this->isDeferred($path)) {
```

Comment:
How is that better?

Reply:
Yes, you only have to check for a value, then use a method.
In such a situation we should always run the most expensive code only if necessary.

Reply:
I see your point.

In this particular case, `defer()` is not expensive and is much more likely to be true, so it makes sense to put it first.